HOW_TO_FIX_A_SECURITY_ISSUE.md

```diff
@@ -0,0 +1,60 @@
+<!--
+ Licensed to the Apache Software Foundation (ASF) under one
+ or more contributor license agreements.  See the NOTICE file
+ distributed with this work for additional information
+ regarding copyright ownership.  The ASF licenses this file
+ to you under the Apache License, Version 2.0 (the
+ "License"); you may not use this file except in compliance
+ with the License.  You may obtain a copy of the License at
+
+   http://www.apache.org/licenses/LICENSE-2.0
+
+ Unless required by applicable law or agreed to in writing,
+ software distributed under the License is distributed on an
+ "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+ KIND, either express or implied.  See the License for the
+ specific language governing permissions and limitations
+ under the License.
+ -->
+
+# Fixing Security Issues in Airflow
+
+Generally speaking, Apache Airflow addresses security issues through the following steps:
+
+## Process
+
+1. **Vulnerability Identification**:
+   The Apache Airflow community actively monitors security channels and mailing lists for reports of vulnerabilities.
+   This includes monitoring security e-mailing lists, forums, and other channels for reports of vulnerabilities.
```

Comment:
Should we add links to those sources?

Reply:
Nice. I do not have all of them handy and not sure if we should add them keeping in mind that we do not expose **all** the details openly

Reply:
Actually, we do not monitor those lists. We got `security@apache.airflow.org` where reporters report us issues.